### PR TITLE
[LIMS-3202] - Merge current HEAD hapi-fhir-starter into Freenome fork

### PR DESCRIPTION
**Pull request by gchwalik-fn**
Requesting to merge recent commits into freenome fork, so that I can include changes to config in branch.

### CODE REVIEW

**Comment by gchwalik-fn** [comment]
Added a few reviewers, but should only need approval from one.

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-# re-downloaded on the next run, unless the pom file changes.
-COPY pom.xml .
-RUN /usr/local/bin/mvn-entrypoint.sh mvn verify clean --fail-never
+ARG HAPI_FHIR_URL=https://github.com/jamesagnew/hapi-fhir/
+ARG HAPI_FHIR_BRANCH=master
+ARG HAPI_FHIR_STARTER_URL=https://github.com/hapifhir/hapi-fhir-jpaserver-starter/
+ARG HAPI_FHIR_STARTER_BRANCH=master
 
-# Copy all of the source code to the image and build it
-COPY . .
```
Comment (jaychia) [inline Dockerfile]: Did this not generate a merge conflict? I'm assuming we want to keep `COPY` instead of merging their `git clone` workflow

Comment (gchwalik-fn) [reply Dockerfile]: I'm hoping to merge in all their recent commits, then create a branch with our working version of config. 

At that point, we could update our master as opposed to keeping it in a separate branch. It seems easier to update from their current first, before applying our changes on top.

Comment (gchwalik-fn) [reply Dockerfile]: I have no current changes in this PR, just hoping to pull in their recent commits.

**Comment by gchwalik-fn** [comment]
Closing this PR as Jay has already checked it off.

**Pull request merged**